internal/operator/subscription.go

```diff
@@ -36,9 +36,6 @@ func (c operatorClient) GetSubscriptionData(catalogSource string, catalogSourceN
 	SubscriptionList := []SubscriptionData{}
 
 	for _, pkgm := range packageManifests.Items {
-		if pkgm.Status.CatalogSource != catalogSource {
```

Comment:
Why is this redundant? This short-circuits to skip the inner loop. Not sure I understand why you would want to take this out?

Reply:
Yes, in the linked issue Edmund points out that `ListPackageManifests` is already filtering by catalogSource

Reply:
which happens up on line 30